src/components/BlipSelect.vue

```diff
@@ -461,6 +465,8 @@ export default {
     },
     select: debounce(
       function(item) {
+        if (this.readonly && !this.editable) return
```

Comment:
Shouldn't return just checking if is `readonly`? Why is needed to check denying `this.editable`?

Reply:
I've considered what @JoaoCMotaJr talked today at the daily. But now with your comment I have some doubts.
If we check the editable, the readonly property will not "override" the behaviour when it is setted true. 
But if we consider that readonly is an optional property and probably only we are going to use it, maybe `&& !this.editable`  it is not necessary

Reply:
I accept both possibilities as valid, as long as we put ourselves in the shoes of whoever will try to figure it out this blip cards next. Therefore, both naming seems contradictable so a new user could deduce as well that both flags does not work together